presto-docs/src/main/sphinx/release/release-0.202.rst

```diff
@@ -0,0 +1,78 @@
+=============
+Release 0.202
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for queries involving aggregations over the result of an outer join
+  (`issue #10592 <https://github.com/prestodb/presto/issues/10592>`_).
+* Fix :func:`map` to fail on duplicate keys, instead of producing a corrupted map silently.
+* Fix OOM in bucketed execution by scheduling new splits on the same worker as
+  the recently finished one.
+* Fix query failure when performing a ``GROUP BY`` on ``JSON`` and ``IPADDRESS`` types.
+* Fix :func:`map_from_entries` to throw ``INVALID_FUNCTION_ARGUMENT`` when input
```

Comment:
"Fix ... to throw" also sounds strange. Also, we don't mention internal error codes in release notes. This could say something like "Make ... fail when ..."